lib/code_corps/policy/helpers.ex

```diff
@@ -67,15 +66,6 @@ defmodule CodeCorps.Policy.Helpers do
 
   def get_organization(_), do: nil
 
-  @doc """
-  Retrieves an organiation invite from a struct, containing a `code` field
-  Returns `CodeCorps.OrganizationInvite` or nil
-  """
-  @spec get_organization_invite(struct) :: OrganizationInvite.t() | nil
-  def get_organization_invite(%{"code" => code}),
-    do: OrganizationInvite |> Repo.get_by(code: code, fulfilled: false)
-  def get_organization_invite(%{}), do: nil
-
```

Comment:
Moved it to `Policy.Organization` since it's the only place it's used in and we don't test ti directly.